meinberlin/apps/budgeting/assets/ControlBar.jsx

```diff
@@ -88,12 +89,12 @@ export const ControlBar = props => {
           )}
           <div className="control-bar__item control-bar__right">
             <button
-              className={
+              className="btn btn--light"
+              aria-label={
                 expandFilters
-                  ? 'btn btn--light active'
```

Comment:
there's no styling for active class on btns

Reply:
@phillimorland hm, i was sure that worked? maybe because of bootstrap. 

In these cases i am not sure what to do though, the design asks for a visual active state (grey background). Now there is no background color anymore right?

Reply:
@khamui btn had active style after click until there was another action on the page (like filter click) then it returns to a non active state and style which was the same behaviour as when the class was added